### ISSUE TITLE
Map button to Undo

### ISSUE THREAD

**Issue by cmbail** [ddvk/remarkable-stylus#13]
I would like the button mapped to undo. It's the one command I tend to need to repeat several times rapidly and is more annoying to have to do through swipes than tool switching or tool/eraser switching.

I couldn't find anything in Readme.md on how to customize the action - is it possible?

**Comment by ddvk** [comment]
undo is double click

**Comment by cmbail** [comment]
TIL. Thank you!

Still a pain if I want to undo 5 strokes. I think I would prefer to have single click mapped to undo. That would require nothing mapped to double-click for undo to work n times. 

I may be rare in that though.

**Comment by jamessui1** [comment]
@ddvk Any way to map the button to erase region instead of current eraser?

**Comment by cmbail** [comment]
After use, I should report that undo mapped to double-click is practically unusable for me. At least half the time I try to undo, I also inadvertently trigger tool switching.

**Comment by ddvk** [comment]
i ll add a toggle or something

**Comment by regrubtaf** [comment]
Thanks for the great added features! I think I've found a bug in the stylus button hack.  When I hold down the button to do a select erase and then release it without lifting the stylus off the tablet, it continues writing and changes the pen thickness.  I noticed it in Marker (goes from thin to thick), but I believe it occurs in other pen settings too.  I haven't noticed this in prior versions of the hack.  Is this how it's intended to function, or is it a bug? Thanks again for all the added features.  I've linked to an [unlisted youtube vid of the possible bug](https://youtu.be/hx0OzY90Cg0).

**Comment by ddvk** [comment]
added as an option in the toggles

**Comment by cmbail** [comment]
Awesome, thank you! I presume these are set in Remarkable's command line, but I couldn't find any documentation in README.md about toggles. I also tried google search on "remarkable-stylus"  "toggles" but it didn't find anything. Can you please help with a link?

**Comment by ddvk** [comment]
in the UI with the toggles, the Lamy Button, but not yet for the rm1

**Comment by cmbail** [comment]
I do have an RM2. But, sorry, there is nothing in my UI.
![RM2_UI](https://user-images.githubusercontent.com/3613239/143782596-fbdc1281-bba8-4da7-8ee2-d20b04885829.jpeg)

**Comment by ddvk** [comment]
did you rerun the script, and also version 2.10.3.379

**Comment by cmbail** [comment]
Yes, I re-ran `sh -c "$(wget https://raw.githubusercontent.com/ddvk/remarkable-stylus/master/patch.sh -O-)"` and the RM2 rebooted.

Confirmed the RM2 is at 2.10.3.379-patch (with ddvk  remarkable-hacks).

I just ran it again. After it rebooted, I see the same UI toggles.

**Comment by ddvk** [comment]
you have to reinstall the remarkable-hacks

**Comment by johnnytravels** [comment]
Any chance the button could be mapped to toggle the highlighter (or previous pen)? Being able to switch between pen and highlighter would be great for reading.

**Comment by suppenkaspar109** [comment]
Previously this worked. But right now I have following situation (with toltec installed & automatic install scripted used)
- hold to erase (works)
- tap to switch between the last two tools (doesn't work)
- double-tap to undo (doesn't work)

**Comment by ddvk** [comment]
there should be an options to change the mode (Lamy Hold Erase is the default, but when you click it there is click undo)

**Comment by koshibb** [comment]
@ddvk I am having a similar type of issue which I am sure PEBKAC....
For the undo, do I need to like point at the item I wish to remove?

**Comment by ddvk** [comment]
> @ddvk I am having a similar type of issue which I am sure PEBKAC.... For the undo, do I need to like point at the item I wish to remove?

nope
